src/main/java/frc/robot/RobotContainer.java

```diff
@@ -40,4 +46,8 @@ public RobotContainer() {
   private void configureBindings() {
     driverController.a().whileTrue(new InstantCommand(intake::deploy, intake));
   }
+
+  public static Command getAutonomousCommand(){
+    return new AutoChargeStationSequence(true, new DriveSubsystem());
```

Comment:
Can you make `true` a named variable in this function just so people don't have to go into AutoChargeStationSequence to see what the true is referring to?